api/views/people.py

```diff
@@ -0,0 +1,44 @@
+from api.share.tools import build_correct_http_response
+from api.share.requests.users import GetUsersCountRequest
+from api.share.requests import InvalidRequestObject
+from api.use_cases.get_users_count import UsersCount
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from http import HTTPStatus
+
+
+class UserCountView(APIView):
+
+    def get(self, request):
+        try:
```

Comment:
this is way too many statements inside a `try` block. Ideally it should contain only one statement

Reply:
Yes, you're right. I think to remove this _try .. expect_ at all but later. It's just in case now. In principle, _except_ part of this block doesn't have to get control at all.

Reply:
I just think that before we merge this into `master`, we should make sure that `try` only tries one statement. We can extract that block into a function if no other changes are possible.

Reply:
I prefer to remove it.